library/Zend/Feed/Abstract.php

```diff
@@ -207,7 +207,7 @@ public function current()
     /**
      * Required by the Iterator interface.
      *
-     * @return mixed The current row number (starts at 0), or NULL if no rows
+     * @return int The current row number (starts at 0), or NULL if no rows
```

Comment:
Hey, i'm brand new in this project, real excited about bringing a bunch of legacy code forward. i hope there's still activity on this.

That said, i'm considering pulling the current submissions into my own fork just to be caught up on the comment tedium.

i happened to see this one and it seems a correction is needed, but i don't know which...
Should it be `@return int|null` as the remark suggests (the property is initialized to `0`) or should the remark drop reference to any `null` return?

Looking forward to contributing.

Reply:
`$this->_entryIndex` is never null, at least from what I see in this class and the @var on the property declaration.
The comment in wrong, not the annotation. (The power of static analysis!)

Anyways, I'll commit a fix.